src/nash/mod.rs

```diff
@@ -207,10 +207,24 @@ impl ExchangeAccount for Nash {
     }
 
     async fn get_all_open_orders(&self) -> Result<Vec<Order>> {
-        let err = MissingImplementationContent {
-            message: String::from("Not supported yet, market paramater is mandatory."),
+        let req = nash_protocol::protocol::list_account_orders::ListAccountOrdersRequest {
+            market: None,
+            before: None,
+            buy_or_sell: None,
+            limit: Some(100),
```

Comment:
Future @notdanilo  @Ejhfast 

I am thinking that we should support Paginators for this request in the future?